packages/databyss-ui/components/Notify/NotifyProvider.js

```diff
@@ -47,7 +43,7 @@ class NotifyProvider extends React.Component {
   constructor(props) {
     super(props)
     startBugsnag(props.options)
-    this.shouldCheckOnlineStatus = props.shouldCheckOnlineStatus
+    // this.shouldCheckOnlineStatus = props.shouldCheckOnlineStatus
```

Comment:
TODO: restore online status checking and inspect pouchdb sync state so we can inform the user of sync progress